docs/src/main/paradox/scala/http/common/json-support.md

```diff
@@ -1,10 +1,11 @@
 # JSON Support
 
-Akka HTTP's @ref[marshalling](marshalling.md) and @ref[unmarshalling](unmarshalling.md) infrastructure makes it rather easy to seamlessly support specific wire representations of your data objects, like JSON, XML or even binary encodings.
+Akka HTTP's @ref[marshalling](marshalling.md) and @ref[unmarshalling](unmarshalling.md) infrastructure makes it rather easy to seamlessly convert application-domain objects from and to JSON.
+Integration with @scala[[spray-json]]@java[[Jackson]] is provided out of the box through the @scala[`akka-http-spray-json`]@java[`akka-http-jackson`] module.
+Integration with other JSON libraries are supported by the community.
+See [the list of current community extensions for Akka HTTP](https://index.scala-lang.org/search?topics=akka-http&topics=json).
```

Comment:
Changed to directly link to scaladex since the community page now simply links there.